java/code/src/com/redhat/rhn/domain/action/ActionFactory.java

```diff
@@ -870,12 +870,11 @@ public static void updateServerActions(Action actionIn, List<Long> serverIds, Ac
         if (log.isDebugEnabled()) {
             log.debug("Action status " + status.getName() + " is going to b set for these servers: " + serverIds);
         }
-        HibernateFactory.getSession()
-                .getNamedQuery("Action.updateServerActions")
-                .setParameter("action_id", actionIn.getId())
-                .setParameter("server_ids", serverIds)
-                .setParameter("status", status.getId())
-                .executeUpdate();
```

Comment:
Is there any particular reason why only this instance of `executeUpdate` was refactored using `udpateByIds`? Could/should we extend the refactoring to other cases?

Reply:
This instance was the one that started the discussion about this work. If you agree, I will extend this refactoring to other cases in a separate PR.